core/src/main/scala/org/apache/spark/ContextCleaner.scala

```diff
@@ -172,18 +172,18 @@ private[spark] class ContextCleaner(
     registerForCleanup(rdd, CleanCheckpoint(parentId))
   }
 
-  /** Register an object for cleanup. */
-  private def registerForCleanup(objectForCleanup: AnyRef, task: CleanupTask): Unit = {
-    referenceBuffer.add(new CleanupTaskWeakReference(task, objectForCleanup, referenceQueue))
-  }
-
   /** Register a SparkListener to be cleaned up when its owner is garbage collected. */
   def registerSparkListenerForCleanup(
       listenerOwner: AnyRef,
       listener: SparkListener): Unit = {
     registerForCleanup(listenerOwner, CleanSparkListener(listener))
   }
 
+  /** Register an object for cleanup. */
+  private def registerForCleanup(objectForCleanup: AnyRef, task: CleanupTask): Unit = {
+    referenceBuffer.add(new CleanupTaskWeakReference(task, objectForCleanup, referenceQueue))
+  }
+
```

Comment:
Hi, @Ngone51 . Could you revert the change in this file?

Reply:
Hi @dongjoon-hyun,  this change intends to let the newly added  `registerSparkListenerForCleanup()` follow the same placement with other `register...ForCleanup` methods. This would make the `ContextCleaner` looks neater. 

This is a style mistake made in the original PR so I fixed it in this follow-up PR. Does it sound ok to you?

Reply:
Got it.